keyboards/sleepy_craft_studios/sleepy_planck/readme.md

```diff
@@ -0,0 +1,27 @@
+# sleepy_planck
+
+![sleepy_planck](https://bucket.sleepycraftstudios.com/images/sleepy_planck_complete_blue_silver_silk-rotated-270.webp)
```

Comment:
Please can this image be (re-)hosted on Imgur?

Reply:
Sure thing